### ISSUE TITLE
random_distortion() not support mask

### ISSUE THREAD

**Issue by duanzhiihao** [mdbloice/Augmentor#173]
Hi, I now have an image(np.array) and a mask(np.array), and I want to perform random distortion on them. I have tried this:
```
assert isinstance(imnp, np.ndarray) and isinstance(gt, np.ndarray)

images = [[imnp,gt]]
p = Augmentor.DataPipeline(images)
p.crop_by_size(1,768,768,centre=False)
p.random_distortion(1,3,3,100)

aug_im = p.sample(1)

# visualize the mask on the image
landmarks = np.argwhere(aug_im[0][1]>=1)
plt.imshow(aug_im[0][0])
plt.scatter(landmarks[:,1], landmarks[:,0], s=10, marker='x', c='g')
plt.pause(0.001)
```
It seems that the distorted mask `aug_im[1]` is not consistent with the distorted image `aug_im [0]`. I can post image if it helps.
So does random_distortion() can't deal with image and its mask at the same time?

**Comment by VincentYCYao** [comment]
@duanzhiihao check issue #188 
you can augment image and mask in parallel using 
```python
p.ground_truth(y_root)
```
**the random distortion fails
random rotation and flipping work fine**

**Comment by mdbloice** [comment]
Hi @VincentYCYao, @duanzhiihao, this has now been resolved as of version 0.2.6. Just upgrade via pip (pip install Augmentor -U)
